### ISSUE TITLE
Issue with API Platform core v2.7.17

### ISSUE THREAD

**Issue by maximehuran** [Sylius/Sylius#16010]
**Sylius version affected**: 1.12.x

**Description**  

While setup the Sylius, I have an error message with autoloader.

```
Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 1
!!
!!  In DebugClassLoader.php line 354:
!!
!!    The autoloader expected class "ApiPlatform\Symfony\Bundle\DependencyInjecti
!!    on\ApiPlatformExtension" to be defined in file ".../vendor/composer/../api-platfo
!!    rm/core/src/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php". T
!!    he file was found but the class was not in it, the class name or namespace
!!    probably has a typo.
!!
!!
```

**Steps to reproduce**  

Run `composer install`

**Possible Solution**  

This commit which introduces the issue : 
https://github.com/api-platform/core/commit/b5a0814788053ff88077b2f13b1fd8a7e6279861

The problem comes from this part of code : 
```
if (!class_exists(\ApiPlatform\Core\Bridge\Symfony\Bundle\ApiPlatformBundle::class)) {
    class_alias(ApiPlatformBundle::class, \ApiPlatform\Core\Bridge\Symfony\Bundle\ApiPlatformBundle::class);
}
```

https://github.com/api-platform/core/blame/v2.7.17/src/Symfony/Bundle/ApiPlatformBundle.php#L59

The version was released the 2024-03-15

**Comment by maximehuran** [comment]
Fron now I locked my app with 2.7.16 version 

![image](https://github.com/Sylius/Sylius/assets/11380627/9c2b6e82-4f38-4720-ba0e-7a2a2bc293a8)

**Comment by Wojdylak** [comment]
Hi @maximehuran 

I close this issue as it has been resolved via PRs #16011 and #16017.
